habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```diff
@@ -164,7 +163,7 @@ public List<String> validatePoetryInstallationAndVersion(ValidationTrackingStatu
         PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
         if (!validationTracker.isAlreadyValidatedPoetryInstallation()) {
             log.debug("Checking if Poetry is installed...");
-            Pair<Boolean, String> poetryInstallStatusAndVersion = poetryHelper.getIsPoetryInstalledAndVersion();
+            Pair<Boolean, String> poetryInstallStatusAndVersion = poetryHelper.getIsPackageManagerInstalledAndVersion();
```

Comment:
should this also be reverted back?
```suggestion
            Pair<Boolean, String> poetryInstallStatusAndVersion = poetryHelper.getIsPoetryInstalledAndVersion();
```
```suggestion
            Pair<Boolean, String> poetryInstallStatusAndVersion = poetryHelper.getIsPackageManagerInstalledAndVersion();
```

Reply:
make sense updated!